### ISSUE TITLE
Unable to get gitstatus on macOS Sierra

### ISSUE THREAD

**Issue by adun** [jaspernbrouwer/powerline-gitstatus#14]
Hi, 

I can't get gitstatus working on macOS Sierra. 
Powerline is working fine and the command `powerline-lint -d` returns nothing after adding Gitstatus segments in config files.

I've installed powerline using the following steps:

- brew install python
- pip install --user powerline-status
- pip install --user powerline-gitstatus
 
I've put the whole configuration files here:
[gist](https://gist.github.com/adun/0003579832655ecd61e084bf941b27ca)
Basically I've taken the default configuration files and added gitstatus segments.

Here is a screenshot of my Terminal after completing the above steps:
![powerline](https://cloud.githubusercontent.com/assets/664059/20990721/9267b882-bcda-11e6-91fa-606247d8ef96.png)

Thank in advance for your help!
adun

**Comment by jaspernbrouwer** [comment]
It looks like you're using bash, and bash doesn't support a right prompt, where the default configuration puts the segment.

Try adding the segment to the `"left"` section of `segments` (not `"right"`) of your configuration:

```json
{
    "segments": {
        "left": [
            ...,
            {
                "function": "powerline_gitstatus.gitstatus",
                "priority": 40
            }
        ],
        "right": []
    }
}
```

**Comment by jaspernbrouwer** [comment]
If that doesn't help, explicitly clear the whole `"right"` section. I'm not sure how stuff put there reacts in bash.

Finally, set the log-level to `DEBUG`, configure a log-file, and look what's appearing there.

**Comment by adun** [comment]
Hi Jasper,

I forgot about `right` section is not supported in `bash`.
I took the [left-only section config file](https://github.com/powerline/powerline/blob/2.5/powerline/config_files/themes/shell/default_leftonly.json) and added the `gitstatus` segment.
Now it's working fine.

Thanks for your answers and the awesome work 👍.
